lexicons/app/bsky/administration/moderationAction.json

```diff
@@ -0,0 +1,21 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.administration.moderationAction",
```

Comment:
Here comes the bikeshed!

Any reason we can't shorten to `.admin.` ?

Reply:
Sounds good 👍